docs/01-app/02-guides/tailwind-v3-css.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: How to install Tailwind CSS v3 in your Next.js application
+nav_title: Tailwind CSS v3
+description: Style your Next.js Application using Tailwind CSS v3 for broader browser support.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+[Tailwind CSS v3](https://v3.tailwindcss.com/) provides broader browser support and uses the traditional JavaScript configuration approach. This guide will walk you through how to install Tailwind CSS v3 in your Next.js application.
```

Comment:
What does this mean? "and uses the traditional JavaScript configuration approach"
```suggestion
[Tailwind CSS v3](https://v3.tailwindcss.com/) provides broader browser support and uses the traditional JavaScript configuration approach. This guide will walk you through how to install Tailwind CSS v3 in your Next.js application.
```

Reply:
The tailwind.cofig.js file - but you know what, I'll just tone it down here and cut the intro short to, `This guide...`